### PR TITLE
feat: Adds big Tooltip component

### PR DESCRIPTION
**Pull request by schne324**
see https://github.com/dequelabs/walnut/issues/1300

### CODE REVIEW

**Comment by schne324** [comment]
Naming is hard...so I went with `variant='big'`. I'm open to something better if anyone can think of anything

**Review by dylanb — APPROVED** [review]

File: packages/styles/line.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.Line {
+  border: none;
+  border-top: 1px solid #ccc;
```
Comment (stephenmathieson) [inline packages/styles/line.css:3]: Should have used variables here.

File: packages/styles/line.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  overflow: visible;
+  margin: 26px auto;
```
Comment (stephenmathieson) [inline packages/styles/line.css:8]: And here

File: packages/styles/line.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+.Line::after {
+  content: '';
+  display: inline-block;
+  margin-top: -4px;
+  width: 7px;
+  height: 7px;
+  border-radius: 50%;
+  background: #fdfdfe;
+  border: 1px solid #ccc;
```
Comment (stephenmathieson) [inline packages/styles/line.css:20]: and here

**Pull request merged**